docs/docs/integrations/tools/databricks.ipynb

```diff
@@ -34,11 +34,19 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Note: you may need to restart the kernel to use updated packages.\n"
+     ]
+    }
```

Comment:
nit: Can we remove this output?

Reply:
I just kept it in because it's what's produced when running the notebook, seems ok to have it show up